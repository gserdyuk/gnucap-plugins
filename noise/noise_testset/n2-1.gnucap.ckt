A simple noise circuit
Vin 1 0 ac 1 dc 0
R1  1 2 50.K
C1  2 3 .15u
RL  3 0 50.K
*.print ac V(1) V(2) V(3)
*.ac  dec 10 10. 10K
*.print noise all
.noise v(3) vin dec 1 10. 10K 
.noise v(3,0) vin dec 1 10. 10K 
.noise v(0,3) vin dec 1 10. 10K 
.noise v(0,3) vIN dec 1 10. 10K 
.end
