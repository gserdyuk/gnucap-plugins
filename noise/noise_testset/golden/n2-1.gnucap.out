Gnucap gnucap-arails  2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
A simple noise circuit 
#Freq inoise_density onoise_density
 10.  5.39007E-15  633.889E-18
 100.  1.695E-15  419.032E-18
 1.E+3  1.65805E-15  414.465E-18
 10.E+3  1.65768E-15  414.419E-18

#index inoise_total onoise_total
0  16.7485E-12  4.15244E-12

#Freq inoise_density onoise_density
 10.  5.39007E-15  633.889E-18
 100.  1.695E-15  419.032E-18
 1.E+3  1.65805E-15  414.465E-18
 10.E+3  1.65768E-15  414.419E-18

#index inoise_total onoise_total
0  16.7485E-12  4.15244E-12

#Freq inoise_density onoise_density
 10.  5.39007E-15  633.889E-18
 100.  1.695E-15  419.032E-18
 1.E+3  1.65805E-15  414.465E-18
 10.E+3  1.65768E-15  414.419E-18

#index inoise_total onoise_total
0  16.7485E-12  4.15244E-12

#Freq inoise_density onoise_density
 10.  5.39007E-15  633.889E-18
 100.  1.695E-15  419.032E-18
 1.E+3  1.65805E-15  414.465E-18
 10.E+3  1.65768E-15  414.419E-18

#index inoise_total onoise_total
0  16.7485E-12  4.15244E-12

